src/Std/Data/HashSet/Basic.lean

```diff
@@ -88,6 +88,13 @@ Equivalent to (but potentially faster than) calling `contains` followed by `inse
   let ⟨replaced, r⟩ := m.inner.containsThenInsertIfNew a ()
   ⟨replaced, ⟨r⟩⟩
 
+/--
+Checks if given key is contained and returns the key if it is, otherwise `none`.
+The result in the `some` case is guaranteed to be pointer equal to the key in the set.
+-/
+@[inline] def getKey? (m : HashSet α) (a : α) : Option α :=
```

Comment:
What do you think, should this be called `get?`?

Reply:
Renamed!